clusterloader2/pkg/util/cluster.go

```diff
@@ -165,11 +167,19 @@ func GetMasterIPs(c clientset.Interface, addressType corev1.NodeAddressType) ([]
 // This code will not be allowed to update to use the node-role label, since
 // node-roles may not be used for feature enablement.
 // DEPRECATED: this will be removed in Kubernetes 1.19
-func LegacyIsMasterNode(nodeName string) bool {
+func LegacyIsMasterNode(node *corev1.Node) bool {
+	// Check labels as an additional signal, fix #1191
+	for key := range node.GetLabels() {
+		if key == keyMasterNodeLabel {
```

Comment:
Check the value too - it can be set to false too.

Reply:
But how to define its value? "false", "False", "No"... It can be set to anything

Reply:
Go the other way. If it's set "node-role.kubernetes.io/master=skfjds", then I wouldn't treat it as master.

Let's change explicitly to:
strings.ToLower(val) == "true"